lib/vcap_env.rb

```diff
@@ -0,0 +1,13 @@
+require 'json'
+
+# Parses ENV['VCAP_SERVICES'] or similar JSON and sets ENV variables for user-provided:credentials
+module VcapEnv
+  def self.set_env(vcap_json)
+    vcap = JSON.parse(vcap_json)
+    env = vcap["user-provided"].find { |service| service["name"] == "compliance-viewer-env"  }
```

Comment:
Well... sinatra_contrib/config-file doesn't help the poultry because it only works in context of `app.rb`... and we need the setting in `config.ru`.  
So, back to adding one more "self-serve" ENV variable, like "OK, set this one variable however you want locally", and back to using `cf set-env` to get the one setting into prod, or maybe encrypting it as another Travis variable (?).
I feel like the above sort of muddies the already-not-crystal-clear-waters, and wonder if thinking of this more as "convention" makes more sense.